text/66-self-managed-stack-sets.md

```diff
@@ -0,0 +1,502 @@
+---
+rfc pr: [#347](https://github.com/aws/aws-cdk-rfcs/pull/347)
+tracking issue: https://github.com/aws/aws-cdk-rfcs/issues/66
+---
+
+# CDK Self Managed Stack Set Support
+As a CDK user, I would like to define stack set infrastructure through native CDK constructs and create/update self-managed stack sets through CDK deploy command.
+
+* **Original Author(s):** @linsona
+* **Tracking Issue**: #66
+* **API Bar Raiser**: @skinny85
+
+## Working Backwards
+
+### CHANGELOG
+- feat(core): New Stack Set core type 
+- feat(assembly): New stack set artifcat type and schema
+- feat(cli): ```cdk deploy``` collects and creates/updates stack set 
+- feat(lib): API Calls/Logic for create/update stack sets
+
+### README
+The ```cdk.StackSet``` construct in CDK defines AWS resources in a stack set and stack set configuration/deployment preferences. This is different from the ```cdk.CfnStackSet``` construct which creates a CFN stack that deploys a CFN stack set. The ```cdk.StackSet``` construct creates the stack set directly if it does not exist, or updates a stack set and existing stack set instances while monitoring the stack set operation.
+
+#### Usage
+To define a stack set, extend the core construct `StackSet`. Then define AWS resources in the same way as defining resources for stacks. Compared to the `Stack` construct, the `StackSet` core construct has different props specific to StackSets. The props define stack set configurations and operation preferences for when updates occur.
+
+**Example:**
+```ts
+class POCStackSet extends cdk.StackSet {
+    constructor(scope: cdk.App, id: string, props: cdk.StackSetProps = {}) {
+        super(scope, id, props);
+
+        new Role(this, 'ExampleRole', {
+          assumedBy: new ServicePrincipal('s3.amazonaws.com')
+        });
+    }
+}
+new POCStackSet(app, 'poc-stack-set',
+    {
+        faultTolerancePercentage: 10,
+        maxConcurrentPercentage: 10,
+        regionConcurrencyType: 'PARALLEL'
+        .
+        .
+        .
+     }
+);
+```
+
+**Deployment:**
+
+To deploy the stack set, the CLI command remains the same: ```cdk deploy```. If the stack set **does not exist**, then CDK will create the stack set directly in CFN. If the stack set **does exist**, then the stack set itself will be updated as well as all **existing** stack set instances. Stack sets instances are added to the stack set outside of the CDK context.
```

Comment:
Let's avoid using "CFN" here. Let's just spell it out as "CloudFormation".

Same comment everywhere in the doc.